js/models/Settings.js

```diff
@@ -26,10 +30,40 @@ export default class extends BaseModel {
   nested() {
     return {
       shippingAddresses: ShippingAddresses,
-      smtpSettings: SMTPSettings,
+      appearanceSettings: AppearanceSettings,
+      transactionSettings: TransactionSettings,
```

Comment:
I don't think there are currently any transaction related keys in the settings API. I'm intending to file an issue with a few additional keys required for the settings API.